composer.json

```diff
@@ -21,12 +21,21 @@
     "php": ">=7.1",
     "ext-PDO": "*",
     "ext-json": "*",
+    "codefog/contao-haste": "^4.17",
```

Comment:
I'm fine with reordering packages. Let's also add `sort-packages: true` to the `config` section, though.

Reply:
Done!